### ISSUE TITLE
topgrade 6.8.0 on macOS: --disable pip3 does not work

### ISSUE THREAD

**Issue by lhaeger** [r-darwish/topgrade#702]
**What did you expect to happen?**

pip3 upgrade should be disabled

**What actually happened?**

pip3 upgrades were not disabled

**Additional details:**
- Which operating system or Linux distribution are you using?

macOS 10.15.7

- How did you install Topgrade?

through MacPorts

<details><pre>
$ topgrade --disable pip3 -v                                                                                                                                                                                       1 ↵
 2021-04-29T16:46:53.540Z DEBUG topgrade::config > Configuration at /Users/lhaeger/.config/topgrade.toml
 2021-04-29T16:46:53.541Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: None, ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, bashit_branch: None, only: None, composer: None, brew: None, linux: None, git: None, windows: None, vagrant: None }
 2021-04-29T16:46:53.542Z DEBUG topgrade         > Version: 6.8.0
 2021-04-29T16:46:53.542Z DEBUG topgrade         > OS: x86_64-apple-darwin
 2021-04-29T16:46:53.542Z DEBUG topgrade         > Args { inner: ["topgrade", "--disable", "pip3", "--disable", "pip3", "-v"] }
 2021-04-29T16:46:53.542Z DEBUG topgrade         > Binary path: Ok("/opt/local/bin/topgrade")
 2021-04-29T16:46:53.542Z DEBUG topgrade         > Self Update: false
 2021-04-29T16:46:53.542Z DEBUG topgrade::utils  > Detected "/opt/local/bin/git" as "git"
 2021-04-29T16:46:53.544Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-04-29T16:46:53.544Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2021-04-29T16:46:53.545Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-04-29T16:46:53.545Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-04-29T16:46:53.545Z DEBUG topgrade::runner > Step "Brew (ARM)"
 2021-04-29T16:46:53.545Z DEBUG topgrade::runner > Step "Brew (Intel)"
 2021-04-29T16:46:53.545Z DEBUG topgrade::runner > Step "Brew Cask (ARM)"
 2021-04-29T16:46:53.545Z DEBUG topgrade::runner > Step "Brew Cask (Intel)"
 2021-04-29T16:46:53.545Z DEBUG topgrade::runner > Step "MacPorts"
 2021-04-29T16:46:53.545Z DEBUG topgrade::utils  > Detected "/opt/local/bin/port" as "port"

―― 18:46:53 - MacPorts ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-29T16:46:53.546Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "port" "selfupdate"
--->  Updating MacPorts base sources using rsync
MacPorts base version 2.6.4 installed,
MacPorts base version 2.6.4 downloaded.
--->  Updating the ports tree
--->  MacPorts base is already the latest version

The ports tree has been updated. To upgrade your installed ports, you should run
  port upgrade outdated
 2021-04-29T16:47:05.766Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "port" "-u" "upgrade" "outdated"
Nothing to upgrade.
--->  Scanning binaries for linking errors
--->  No broken files found.
--->  No broken ports found.
 2021-04-29T16:47:07.743Z DEBUG topgrade::runner   > Step "Microsoft AutoUpdate"
 2021-04-29T16:47:07.743Z DEBUG topgrade::utils    > Path "/Library/Application Support/Microsoft/MAU2.0/Microsoft AutoUpdate.app/Contents/MacOS/msupdate" exists

―― 18:47:07 - Microsoft AutoUpdate ―――――――――――――――――――――――――――――――――――――――――――――
 2021-04-29T16:47:07.743Z DEBUG topgrade::executor > Running "/Library/Application Support/Microsoft/MAU2.0/Microsoft AutoUpdate.app/Contents/MacOS/msupdate" "--list"
Checking for updates...
No updates available
 2021-04-29T16:47:13.998Z DEBUG topgrade::executor > Running "/Library/Application Support/Microsoft/MAU2.0/Microsoft AutoUpdate.app/Contents/MacOS/msupdate" "--install"
Detecting and downloading updates...
No updates applied
 2021-04-29T16:47:22.972Z DEBUG topgrade::runner   > Step "yadm"
 2021-04-29T16:47:22.972Z DEBUG topgrade::runner   > Step "nix"
 2021-04-29T16:47:22.972Z DEBUG topgrade::runner   > Step "home-manager"
 2021-04-29T16:47:22.972Z DEBUG topgrade::runner   > Step "asdf"
 2021-04-29T16:47:22.973Z DEBUG topgrade::utils    > Path "/Users/lhaeger/.config/emacs" doesn't exist
 2021-04-29T16:47:22.973Z DEBUG topgrade::utils    > Path "/Users/lhaeger/.emacs.d" doesn't exist
 2021-04-29T16:47:22.973Z DEBUG topgrade::steps::git > /Users/lhaeger/.doom.d does not exists
 2021-04-29T16:47:22.973Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.vim is a git repository
 2021-04-29T16:47:23.001Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-04-29T16:47:23.001Z DEBUG topgrade::steps::git > /Users/lhaeger/.config/nvim does not exists
 2021-04-29T16:47:23.001Z DEBUG topgrade::steps::git > /Users/lhaeger/.zshrc is a file. Checking /Users/lhaeger
 2021-04-29T16:47:23.001Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger is a git repository
 2021-04-29T16:47:23.010Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-04-29T16:47:23.010Z DEBUG topgrade::steps::git > /Users/lhaeger/.tmux does not exists
 2021-04-29T16:47:23.010Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.config/fish is a git repository
 2021-04-29T16:47:23.020Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-04-29T16:47:23.020Z DEBUG topgrade::steps::git > /Users/lhaeger/Library/Application Support/openbox does not exists
 2021-04-29T16:47:23.020Z DEBUG topgrade::steps::git > /Users/lhaeger/Library/Application Support/bspwm does not exists
 2021-04-29T16:47:23.020Z DEBUG topgrade::steps::git > /Users/lhaeger/Library/Application Support/i3 does not exists
 2021-04-29T16:47:23.020Z DEBUG topgrade::steps::git > /Users/lhaeger/Library/Application Support/sway does not exists
 2021-04-29T16:47:23.020Z DEBUG topgrade::runner     > Step "Git repositories"
 2021-04-29T16:47:23.020Z DEBUG topgrade::runner     > Step "zr"
 2021-04-29T16:47:23.020Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.020Z DEBUG topgrade::runner     > Step "antibody"
 2021-04-29T16:47:23.020Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.020Z DEBUG topgrade::runner     > Step "antigen"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Path "/Users/lhaeger/.zshrc" exists
 2021-04-29T16:47:23.021Z DEBUG topgrade::runner     > Step "zplug"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Path "/Users/lhaeger/.zshrc" exists
 2021-04-29T16:47:23.021Z DEBUG topgrade::runner     > Step "zinit"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Path "/Users/lhaeger/.zshrc" exists
 2021-04-29T16:47:23.021Z DEBUG topgrade::runner     > Step "zim"
 2021-04-29T16:47:23.021Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.043Z TRACE topgrade::executor   > Output of "zsh" "-c" "[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2021-04-29T16:47:23.043Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2021-04-29T16:47:23.043Z DEBUG topgrade::utils      > Detected "/opt/local/bin/zsh" as "zsh"
 2021-04-29T16:47:23.043Z DEBUG topgrade::utils      > Path "/Users/lhaeger/.oh-my-zsh" exists

―― 18:47:23 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-29T16:47:23.053Z TRACE topgrade::executor   > Output of "zsh" "-c" "test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2021-04-29T16:47:23.053Z DEBUG topgrade::steps::zsh > Running zsh returned exit code: 1: . Using default path: /Users/lhaeger/.oh-my-zsh/custom
 2021-04-29T16:47:23.053Z DEBUG topgrade::steps::zsh > oh-my-zsh custom dir: /Users/lhaeger/.oh-my-zsh/custom
 2021-04-29T16:47:23.053Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom is a git repository
 2021-04-29T16:47:23.062Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
 2021-04-29T16:47:23.062Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom/plugins is a git repository
 2021-04-29T16:47:23.070Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
 2021-04-29T16:47:23.070Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom/plugins/zsh-autosuggestions is a git repository
 2021-04-29T16:47:23.079Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh/custom/plugins/zsh-autosuggestions\n", stderr: "" }
 2021-04-29T16:47:23.079Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom/plugins/example is a git repository
 2021-04-29T16:47:23.087Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
 2021-04-29T16:47:23.087Z DEBUG topgrade::steps::git > /Users/lhaeger/.oh-my-zsh/custom/example.zsh is a file. Checking /Users/lhaeger/.oh-my-zsh/custom
 2021-04-29T16:47:23.087Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom is a git repository
 2021-04-29T16:47:23.095Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
 2021-04-29T16:47:23.095Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom/themes is a git repository
 2021-04-29T16:47:23.103Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
 2021-04-29T16:47:23.103Z DEBUG topgrade::steps::git > /Users/lhaeger/.oh-my-zsh/custom/themes/example.zsh-theme is a file. Checking /Users/lhaeger/.oh-my-zsh/custom/themes
 2021-04-29T16:47:23.103Z DEBUG topgrade::steps::git > Checking if /Users/lhaeger/.oh-my-zsh/custom/themes is a git repository
 2021-04-29T16:47:23.110Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/lhaeger/.oh-my-zsh\n", stderr: "" }
Pulling custom plugins and themes
 2021-04-29T16:47:23.118Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2021-04-29T16:47:23.127Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ae315ded4dba10685dbbafbfa2ff3c1aefeb490d\n", stderr: "" }
Pulling /Users/lhaeger/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2021-04-29T16:47:23.904Z TRACE topgrade::executor   > Output of "/opt/local/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ae315ded4dba10685dbbafbfa2ff3c1aefeb490d\n", stderr: "" }
Up-to-date /Users/lhaeger/.oh-my-zsh/custom/plugins/zsh-autosuggestions
 2021-04-29T16:47:23.905Z DEBUG topgrade::executor   > Running "zsh" "/Users/lhaeger/.oh-my-zsh/tools/upgrade.sh"
Updating Oh My Zsh
From https://github.com/ohmyzsh/ohmyzsh
 * branch            master     -> FETCH_HEAD
Current branch master is up to date.
         __                                     __
  ____  / /_     ____ ___  __  __   ____  _____/ /_
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / /
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/
                        /____/

Oh My Zsh is already at the latest version.
To keep up with the latest news and updates, follow us on Twitter: https://twitter.com/ohmyzsh
Want to get involved in the community? Join our Discord: https://discord.gg/ohmyzsh
Get your Oh My Zsh swag at: https://shop.planetargon.com/collections/oh-my-zsh
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "fisher"
 2021-04-29T16:47:24.978Z DEBUG topgrade::utils      > Detected "/usr/local/bin/fish" as "fish"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "bash-it"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2021-04-29T16:47:24.978Z DEBUG topgrade::utils      > Detected "/usr/local/bin/fish" as "fish"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "tmux"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "TLDR"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "pearl"
 2021-04-29T16:47:24.978Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2021-04-29T16:47:24.979Z DEBUG topgrade::utils      > Detected "/opt/local/bin/bash" as "bash"
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step "apm"
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step "vscode"
 2021-04-29T16:47:24.979Z DEBUG topgrade::utils      > Detected "/opt/local/bin/code" as "code"

―― 18:47:24 - Visual Studio Code ―――――――――――――――――――――――――――――――――――――――――――――――
This step has been removed from Topgrade due to its unreliability
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step "vscodium"
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step "fossil"
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step "rustup"
 2021-04-29T16:47:24.979Z DEBUG topgrade::runner     > Step ".NET"
 2021-04-29T16:47:24.979Z DEBUG topgrade::utils      > Detected "/usr/local/share/dotnet/dotnet" as "dotnet"
 2021-04-29T16:47:25.274Z DEBUG topgrade::runner     > Step "choosenim"
 2021-04-29T16:47:25.274Z DEBUG topgrade::runner     > Step "cargo"
 2021-04-29T16:47:25.274Z DEBUG topgrade::utils      > Detected "/opt/local/bin/cargo" as "cargo"
 2021-04-29T16:47:25.274Z DEBUG topgrade::runner     > Step "Flutter"
 2021-04-29T16:47:25.274Z DEBUG topgrade::runner     > Step "Emacs"
 2021-04-29T16:47:25.274Z DEBUG topgrade::runner     > Step "opam"
 2021-04-29T16:47:25.275Z DEBUG topgrade::runner     > Step "vcpkg"
 2021-04-29T16:47:25.275Z DEBUG topgrade::runner     > Step "pipx"
 2021-04-29T16:47:25.275Z DEBUG topgrade::runner     > Step "pip3"
 2021-04-29T16:47:25.275Z DEBUG topgrade::utils      > Detected "/usr/bin/pip3" as "pip3"

―― 18:47:25 - pip3 ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-29T16:47:25.275Z DEBUG topgrade::executor   > Running "/usr/bin/pip3" "install" "--upgrade" "--user" "pip"
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
Requirement already satisfied: pip in ./Library/Python/3.8/lib/python/site-packages (21.1)
WARNING: Value for scheme.platlib does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/lhaeger/Library/Python/3.8/lib/python/site-packages
sysconfig: /Users/lhaeger/Library/Python/3.8/lib/python3.8/site-packages
WARNING: Value for scheme.purelib does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/lhaeger/Library/Python/3.8/lib/python/site-packages
sysconfig: /Users/lhaeger/Library/Python/3.8/lib/python3.8/site-packages
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /Users/lhaeger/Library/Python/3.8/include/python3.8/UNKNOWN
sysconfig: /Users/lhaeger/Library/Python/3.8/include/python3.8
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
 2021-04-29T16:47:26.402Z DEBUG topgrade::runner     > Step "stack"
 2021-04-29T16:47:26.402Z DEBUG topgrade::runner     > Step "tlmgr"
 2021-04-29T16:47:26.403Z DEBUG topgrade::runner     > Step "myrepos"
 2021-04-29T16:47:26.403Z DEBUG topgrade::runner     > Step "jetpack"
 2021-04-29T16:47:26.403Z DEBUG topgrade::runner     > Step "vim"
 2021-04-29T16:47:26.403Z DEBUG topgrade::utils      > Detected "/opt/local/bin/vim" as "vim"
 2021-04-29T16:47:26.422Z TRACE topgrade::executor   > Output of "/opt/local/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Apr  6 2021 16:23:58)\nmacOS version - x86_64\nIncluded patches: 1-2681\nCompiled by MacPorts\nHuge version without GUI.  Features included (+) or not (-):\n+acl               -farsi             +mouse_sgr         +tag_binary\n+arabic            +file_in_path      -mouse_sysmouse    -tag_old_static\n+autocmd           +find_in_path      +mouse_urxvt       -tag_any_white\n+autochdir         +float             +mouse_xterm       -tcl\n-autoservername    +folding           +multi_byte        +termguicolors\n-balloon_eval      -footer            +multi_lang        +terminal\n+balloon_eval_term +fork()            -mzscheme          +terminfo\n-browse            +gettext           +netbeans_intg     +termresponse\n++builtin_terms    -hangul_input      +num64             +textobjects\n+byte_offset       +iconv             +packages          +textprop\n+channel           +insert_expand     +path_extra        +timers\n+cindent           +ipv6              -perl              +title\n-clientserver      +job               +persistent_undo   -toolbar\n+clipboard         +jumplist          +popupwin          +user_commands\n+cmdline_compl     +keymap            +postscript        +vartabs\n+cmdline_hist      +lambda            +printer           +vertsplit\n+cmdline_info      +langmap           +profile           +virtualedit\n+comments          +libcall           -python            +visual\n+conceal           +linebreak         -python3           +visualextra\n+cryptv            +lispindent        +quickfix          +viminfo\n-cscope            +listcmds          +reltime           +vreplace\n+cursorbind        +localmap          +rightleft         +wildignore\n+cursorshape       -lua               -ruby              +wildmenu\n+dialog_con        +menu              +scrollbind        +windows\n+diff              +mksession         +signs             +writebackup\n+digraphs          +modify_fname      +smartindent       -X11\n-dnd               +mouse             -sound             -xfontset\n-ebcdic            -mouseshape        +spell             -xim\n+emacs_tags        +mouse_dec         +startuptime       -xpm\n+eval              -mouse_gpm         +statusline        -xsmp\n+ex_extra          -mouse_jsbterm     -sun_workshop      -xterm_clipboard\n+extra_search      +mouse_netterm     +syntax            -xterm_save\n   system vimrc file: \"/opt/local/etc/vimrc\"\n     user vimrc file: \"$HOME/.vimrc\"\n 2nd user vimrc file: \"~/.vim/vimrc\"\n      user exrc file: \"$HOME/.exrc\"\n       defaults file: \"$VIMRUNTIME/defaults.vim\"\n  fall-back for $VIM: \"/opt/local/share/vim\"\nCompilation: /usr/bin/clang -c -I. -Iproto -DHAVE_CONFIG_H -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -DMACOS_X -DMACOS_X_DARWIN -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -arch x86_64 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 \nLinking: /usr/bin/clang -L/opt/local/lib -Wl,-headerpad_max_install_names -Wl,-syslibroot,/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -arch x86_64 -o vim -lm -lncurses -liconv -lintl -framework AppKit \n", stderr: "" }
 2021-04-29T16:47:26.422Z DEBUG topgrade::runner     > Step "Neovim"
 2021-04-29T16:47:26.422Z DEBUG topgrade::runner     > Step "voom"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "npm"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "yarn"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "deno"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "composer"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "krew"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "gem"
 2021-04-29T16:47:26.423Z DEBUG topgrade::utils      > Detected "/usr/bin/gem" as "gem"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "sheldon"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "rtcl"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "gcloud"
 2021-04-29T16:47:26.423Z DEBUG topgrade::runner     > Step "micro"
 2021-04-29T16:47:26.424Z DEBUG topgrade::runner     > Step "raco"
 2021-04-29T16:47:26.424Z DEBUG topgrade::runner     > Step "App Store"
 2021-04-29T16:47:26.424Z DEBUG topgrade::runner     > Step "System upgrade"

―― 18:47:26 - macOS system update ――――――――――――――――――――――――――――――――――――――――――――――
Finding available software
 2021-04-29T16:47:35.681Z DEBUG topgrade::steps::os::macos > Output { status: ExitStatus(ExitStatus(0)), stdout: "Software Update Tool\n\nFinding available software\n", stderr: "No new software available.\n" }
 2021-04-29T16:47:35.681Z DEBUG topgrade::steps::os::macos > "No new software available.\n"
No new software available.

―― 18:47:35 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Brew (ARM): SKIPPED: Cannot find "/opt/homebrew/bin/brew" in PATH
Brew (Intel): SKIPPED: Cannot find "/usr/local/bin/brew" in PATH
Brew Cask (ARM): SKIPPED: Cannot find "/opt/homebrew/bin/brew" in PATH
Brew Cask (Intel): SKIPPED: Cannot find "/usr/local/bin/brew" in PATH
MacPorts: OK
Microsoft AutoUpdate: OK
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antigen: SKIPPED: Path "/Users/lhaeger/antigen.zsh" doesn't exist
zplug: SKIPPED: Path "/Users/lhaeger/.zplug" doesn't exist
zinit: SKIPPED: Path "/Users/lhaeger/.zinit" doesn't exist
zim: SKIPPED: Path "/Users/lhaeger/.zim" doesn't exist
oh-my-zsh: OK
fisher: SKIPPED: Path "/Users/lhaeger/.config/fish/functions/fisher.fish" doesn't exist
bash-it: SKIPPED: Path "/Users/lhaeger/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Path "/Users/lhaeger/.local/share/omf/pkg/omf/functions/omf.fish" doesn't exist
tmux: SKIPPED: Path "/Users/lhaeger/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: SKIPPED: Cannot find "tldr" in PATH
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/Users/lhaeger/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
vscode: OK
vscodium: SKIPPED: Cannot find "codium" in PATH
fossil: SKIPPED: Cannot find "fossil" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: No dotnet global tools installed
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Path "/Users/lhaeger/.cargo/.crates.toml" doesn't exist
Flutter: SKIPPED: Cannot find "flutter" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
pip3: OK
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: Cannot find "tlmgr" in PATH
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Path "/Users/lhaeger/.vim/vimrc" doesn't exist
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: Cannot find "npm" in PATH
yarn: SKIPPED: Cannot find "yarn" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Path "/Users/lhaeger/.gem" doesn't exist
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
micro: SKIPPED: Cannot find "micro" in PATH
raco: SKIPPED: Cannot find "raco" in PATH
App Store: SKIPPED: Cannot find "mas" in PATH
System upgrade: OK
 2021-04-29T16:47:35.682Z DEBUG topgrade::terminal         > Desktop notification: Topgrade finished sucessfully
</pre></details>

**Comment by Mrfiregem** [comment]
I think this is already fixed (#689) but a new version hasn't been released yet.

**Comment by r-darwish** [comment]
Please reopen if it happens then the next version comes out.
